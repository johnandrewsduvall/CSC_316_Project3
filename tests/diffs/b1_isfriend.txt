1,4d0
< isfriend A B
< isfriend H I
< isfriend A G
< isfriend E I
