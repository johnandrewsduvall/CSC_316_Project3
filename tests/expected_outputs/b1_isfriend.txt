isfriend A B
isfriend H I
isfriend A G
isfriend E I
